tests/integration/server.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python
+
+from google.cloud import logging as gcloud_logging
+from google.cloud import monitoring as gcloud_monitoring
+from google.cloud.monitoring import MetricKind, ValueType
+from google.cloud.exceptions import Forbidden as ForbiddenException
+from google.cloud.exceptions import NotFound as NotFoundException
+from oauth2client.client import GoogleCredentials
+
+import logging
+import time
+
+from flask import Flask, request, jsonify
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello_world():
+	return 'Hello World!'
+
+
+@app.route('/logging', methods=['POST'])
+def _logging():
+	request_data = request.get_json()
+	if request_data is None:
+		raise ErrorResponse("Unable to parse request JSON: did you set the Content-type header?")
```

Comment:
Use single-quotes throughout. Also, wrap line length to 79 characters throughout.